src/clean.c

```diff
@@ -189,6 +189,27 @@ static bool is_all_digits(const char *s)
 	return true;
 }
 
+static bool is_all_xdigits(const char *s)
+{
+	for (; *s; s++) {
+		if (!isxdigit(toupper(*s))) {
+			return false;
+		}
+	}
+	return true;
+}
+
+static bool is_hashed_manifest(const char UNUSED_PARAM *dir, const struct dirent *entry)
+{
+	static const char prefix[] = "Manifest.";
+	static const size_t prefix_len = sizeof(prefix) - 1;
```

Comment:
Any reason for using static storage here? `prefix_len` will be recomputed every time from `prefix`, so I'm not seeing why it's needed.

Reply:
If you set it as static it will compute it only once. (I have the felling the compiler will optimize that anyway, but setting this as static makes sense to me)